Sources/StreamChatUI/Composer/ComposerVC.swift

```diff
@@ -228,6 +229,9 @@ open class _ComposerVC<ExtraData: ExtraDataTypes>: _ViewController,
         super.setUp()
 
         composerView.inputMessageView.textView.delegate = self
+        
+        // Set the delegate for handling the pasting of UIImages in the text view
+        composerView.inputMessageView.textView.imagePasteDelegate = self
```

Comment:
We probably going to handle more attachment types, like pasting videos, etc.. we should try to make the delegate a bit more generic 👍